Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -31,8 +31,8 @@ public record struct Registry
     private const string DockerManifestListV2 = "application/vnd.docker.distribution.manifest.list.v2+json";
     private const string DockerContainerV1 = "application/vnd.docker.container.image.v1+json";
 
-    private readonly Uri BaseUri;
-    private readonly string RegistryName => BaseUri.Host;
+    public readonly Uri BaseUri;
+    private readonly string RegistryName => BaseUri.GetComponents(UriComponents.HostAndPort, UriFormat.Unescaped);
```

Comment:
This is used only for documentation - it definitely makes sense for the localhost test registry container, but if the port isn't explicitly specified in the Uri, does it come through here? if so, we should change this.